packages/just-scripts/src/stack/lockfile.ts

```diff
@@ -0,0 +1,20 @@
+import { getAvailableStacks } from './getAvailableStacks';
+import path from 'path';
+import fs from 'fs';
+
+export function writeLockFile(rootPath: string) {
```

Comment:
What does the lock file do? Should it be checked in?

Reply:
oh now I see at the end of the PR that you have a log message about that, but comments would be good too

Reply:
Should be checked in. I'm open to some suggestion here. The flow is that I would like people to update their package.json with the intention to update the devDep of the stack, then run "npm run upgrade-repo" on their own. That required us to have known a last upgraded version somewhere - I don't know of a really good way other than making a file change somewhere as part of the upgrade process.